.github/workflows/gen.yml

```diff
@@ -4,12 +4,6 @@ on:
     branches: [2.*, 3.*, 4.*, main]
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths:
```

Comment:
We don't want to run Generate if just some random test file has changed

Reply:
Generate is required. So we need this.